### PR TITLE
ADDED DB2 Handler 

### LINKED ISSUE #1170: Support DB2 database integration
Support IBM DB2 database integration 

We need to figure out how to connect, to enable AI-Tables or deploy in DB2 container or DB2 on Cloud

P.D. An apology for incomplete or not answering issue: https://github.com/mindsdb/mindsdb/issues/1167#event-4489203966

### PR DESCRIPTION
**Pull request by parthiv11**
Fixes #1170 

## Please describe what changes you made, in as much detail as possible
  - Adde db2 handler using **`ibm_db_dbi`** driver.

- **NOTE** : _Suggest if some modification to be done_

@mindsdbadmin

### CODE REVIEW

**Comment by parthiv11** [comment]
**@ZoranPandovski** and _@StpMax_   i have made  tried to use my knowledge to add **DB2Handler**.

-` Please Review and Accept it before deadline `

**Comment by parthiv11** [comment]
Hey @mindsdbadmin @ZoranPandovski  I have completed DB2 integration and it is working !!

* Of course at last day but completed before deadline  Hhaaa....
* You can check and test  it out and merge !!!!!

**Thank you to everyone who helped me 👍💜**

0create

1use


2show


3 select

4predictor

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

File: mindsdb/integrations/handlers/db2_handler/db2_handler.py [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+connection_args_example = OrderedDict(
+    host='127.0.0.1',
+    port='25000',
+    password='1234',
+    user='db2admin',
+    schemaName="db2admin",
+    database="BOOKS",
+)
+
+# 💙💙💛💛💜💜🤍🤎🤍🤍🤎🤎🤎❤️❤️❤️❤️❤️🧡🧡🧡💜💜❤️❤️❤️🤍🤎🤍🤍❤️🧡💙💙
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/db2_handler/db2_handler.py:301]: Please just remove this, and I will merge. I've tried CREATE PREDICTOR  too and everything works

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

File: mindsdb/integrations/handlers/db2_handler/db2_handler.py (hunk 1 above)
Comment (parthiv11) [reply mindsdb/integrations/handlers/db2_handler/db2_handler.py]: removed it.

But I think it looks nice ;-}

**Comment by parthiv11** [comment]
@ZoranPandovski you can merge This PR now

**Review by ZoranPandovski — APPROVED** [review]

**Comment by parthiv11** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   ***I have read the CLA Document and I hereby sign the CLA***
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by parthiv11** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**